storage/storage-client/src/lib.rs

```diff
@@ -515,18 +515,29 @@ pub trait StorageWrite: Send + Sync {
     ) -> Result<()>;
 }
 
-pub struct StorageReaderWithRuntimeHandle {
-    reader: Arc<dyn StorageRead>,
-    rt_handle: Handle,
+pub struct StorageClientWithRuntimeHandle {
+    reader: Arc<StorageReadServiceClient>,
+    writer: Arc<StorageWriteServiceClient>,
+    rt: Runtime,
```

Comment:
Is it intended to be a handle or a runtime?

Reply:
ugh, it's intentional, because if `Runtime` ends it's lifetime, `Handle`s get invalidated. Had to change to carry `Runtime`s to make tests work.